airgun/entities/media.py

```diff
@@ -0,0 +1,83 @@
+from navmazing import NavigateToSibling
+
+from airgun.entities.base import BaseEntity
+from airgun.navigation import NavigateStep, navigator
+from airgun.views.media import (
+    MediaCreateView,
+    MediaEditView,
+    MediasView,
+)
+
+
+class MediaEntity(BaseEntity):
+
+    def create(self, values):
+        """Create new media"""
+        view = self.navigate_to(self, 'New')
+        view.fill(values)
+        view.submit.click()
+        view.flash.assert_no_error()
+        view.flash.dismiss()
+
+    def search(self, value):
+        """Search for specific media"""
+        view = self.navigate_to(self, 'All')
+        return view.search(value)
+
+    def read(self, entity_name, widget_names=None):
+        """Read values for existing media"""
+        view = self.navigate_to(self, 'Edit', entity_name=entity_name)
+        return view.read(widget_names=widget_names)
+
+    def update(self, entity_name, values):
+        """Update media values"""
+        view = self.navigate_to(self, 'Edit', entity_name=entity_name)
+        view.fill(values)
+        view.submit.click()
+        view.flash.assert_no_error()
+        view.flash.dismiss()
+
+    def delete(self, entity_name):
+        """Delete media"""
+        view = self.navigate_to(self, 'All')
+        view.search(entity_name)
+        view.table.row(name=entity_name)['Actions'].widget.click(
+            handle_alert=True)
+
+
+@navigator.register(MediaEntity, 'All')
+class ShowAllMedias(NavigateStep):
```

Comment:
Maybe we should use 'Medium' instead of 'Medias' as a plural form?

Reply:
whatever